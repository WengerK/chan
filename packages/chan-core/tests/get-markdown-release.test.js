import toVFile from 'to-vfile'
import { dirname } from 'dirname-filename-esm'

import { getMarkdownRelease } from '../src/index.js'

test('get markdown release', async () => {
  const release = await getMarkdownRelease(toVFile.readSync(`${dirname(import.meta)}/__files__/used.md`), {
    version: '0.0.4',
  })

  expect(release).toMatchSnapshot()
})

