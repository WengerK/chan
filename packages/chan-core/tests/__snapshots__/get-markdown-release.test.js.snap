// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get markdown release 1`] = `
"## [0.0.4] - 2014-08-09
### Added
- Better explanation of the difference between the file (\\"CHANGELOG\\")
  and its function \\"the change log\\".

### Changed
- Refer to a \\"change log\\" instead of a \\"CHANGELOG\\" throughout the site
  to differentiate between the file and the purpose of the file â€” the
  logging of changes.

### Removed
- Remove empty sections from CHANGELOG, they occupy too much space and
  create too much noise in the file. People will have to assume that the
  missing sections were intentionally left out because they contained no
  notable changes.

[0.0.4]: https://github.com/olivierlacan/keep-a-changelog/compare/v0.0.3...v0.0.4
"
`;
